projects/1.19/assets/fabric-seasons/seasons/lang/zh_cn.json

```
@@ -0,0 +1,21 @@
+{
+  "tooltip.seasons.spring": "春",
+  "tooltip.seasons.summer": "夏",
+  "tooltip.seasons.fall": "秋",
+  "tooltip.seasons.winter": "冬",
+  "tooltip.seasons.faster_grow": "在此季节生长更快",
+  "tooltip.seasons.normal_grow": "在此季节正常生长",
+  "tooltip.seasons.slowed_grow": "在此季节生长更慢",
+  "tooltip.seasons.not_grow": "在此季节不会生长",
+  "tooltip.seasons.show_more": "按%s显示更多信息",
+
+  "chat.seasons.mod_installed": "看来你已经安装了%s！",
+  "chat.seasons.compatibility": "安装拓展可完全兼容Fabric Seasons。",
+  "chat.seasons.mod_not_installed": "看来你没有安装%s！",
+  "chat.seasons.extras": "安装附属来获得自定义的物品和方块来帮助你的冒险。",
```

```suggestion
  "chat.seasons.extras": "安装附属以获得自定义物品和方块来帮助你的冒险。",
```
原文就有点怪怪的，但是也不知道怎么改，就这样吧（）

---

en (๑ᵔ⤙ᵔ๑)